consensus/chain_selector.go

```diff
@@ -16,6 +16,21 @@ import (
 	"github.com/filecoin-project/go-filecoin/types"
 )
 
+// NewECV is the constant V defined in the EC spec.
+const NewECV uint64 = 2
+
+// PI is the multiplicand in the null penalty term
+const PI float64 = 0.87
+
+// NullThresh is the min number of null rounds before the penalty kicks in
+const NullThresh = 3
+
+// ECV is the constant V defined in the EC spec.
+const ECV uint64 = 10
```

Comment:
I initially thought this was unused, but realise it's there for the old function. Please clearly separate the constants for the old and new functions.